src/components/SelectComponents/AutosuggestMulti/AutosuggestMulti.tsx

```diff
@@ -164,6 +168,22 @@ export function AutosuggestMulti<S>(props: Props<S>) {
         </div>
     );
 
+    const renderList: ListRendererHelper<S> = (listProps) => {
+        const { inputValue: inputToList } = listProps;
```

Comment:
Unexpected 'No suggestion' placeholder still gets displayed after selecting a suggestion.

Reply:
was a bug in the story. Should be fine now